src/server/typingsCache.ts

```diff
@@ -24,7 +24,7 @@ namespace ts.server {
         globalTypingsCacheLocation: undefined
     };
 
-    class TypingsCacheEntry {
+    interface TypingsCacheEntry {

```

Comment:
so why did we make this change?

Reply:
we were never doing ```new TypingsCacheEntry()``` so i seems like a interface rather than class